README.md

```diff
@@ -6,6 +6,57 @@ See it in action in the [react-admin](https://marmelab.com/react-admin/) [demo](
 
 ## Usage
 
+### MSW
+
+We recommend you use [MSW](https://mswjs.io/) to mock your API. This will allow you to inspect requests as you usually do in the developer devtool network tab.
```

Comment:
```suggestion
We recommend you use [MSW](https://mswjs.io/) to mock your API. This will allow you to inspect requests as you usually do in the devtools network tab.
```